test/Microsoft.TestPlatform.TestHostProvider.UnitTests/Hosting/DefaultTestHostManagerTests.cs

```diff
@@ -383,6 +384,7 @@ public void LaunchTestHostShouldUseCustomHostIfSet()
         }
 
         [TestMethod]
+        [Ignore]
         public void LaunchTestHostShouldSetExitCallbackInCaseCustomHost()
         {
             var mockCustomLauncher = new Mock<ITestHostLauncher>();
```

Comment:
Why are these tests skipped? Do you plan to resurrect them?

Reply:
Done, tests were resurrected.